pages/index.tsx

```diff
@@ -11,7 +11,7 @@ export const getServerSideProps = async(context: any) => {
 
   return {
     redirect: {
-      destination: session ? '/menu' : '/auth',
+      destination: session ? '/menu' : '/intro',
```

Comment:
How we'll we go from intro to auth?

Reply:
con el evento click que se le declaro en el intro.tsx